src/cloud-api-adaptor/cmd/agent-protocol-forwarder/keys.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"log"
```

Comment:
Consider using `logrus` or similar so that you can indicate if a given log entry is info, warning, error, etc.

Reply:
I agree that using the standard golang log leave much to be desired. 
However, this is what this repository is using at the cloud-api-adaptor and at the forwarder and I align to what is already there. 

I agree that it makes sense to move the entire repository to using more advanced log package - we can open an issue for that and maybe advance toward it in a future PR.

I would not wish to have one log package used by secure-comms and a different one in the rest of the code...